packages/react-experiments/src/components/PersonaCoin/PersonaCoin.view.tsx

```diff
@@ -43,7 +44,7 @@ export const PersonaCoinView: IPersonaCoinComponent['view'] = props => {
         imageShouldStartVisible={props.imageShouldStartVisible}
         imageAlt={props.imageAlt}
       />
-      <Slots.presence coinSize={coinSize} size={coinSize} />
+      <Slots.presence coinSize={coinSize} size={coinSize as PersonaSize} />
```

Comment:
This seems like it might have been a bug, but my goal with these changes is to only fix type errors and not change semantics.